Documentation/deployment-and-configuration.md

```diff
@@ -46,6 +46,39 @@ etcd_certfile=/etc/ssl/etcd/client.pem
 etcd_keyfile=/etc/ssl/etcd/client-key.pem
 ```
 
+### Basic Authentication
+
+If your etcd cluster has [Basic authentication][etcd-authentication] enabled, you will need to configure fleet to use an username/password combination for a valid user in the system. *Authentication is only available since etcd 2.1.X and greater.*
+The examples below show how to achieve this:
+
+#### Using systemd Drop-Ins
+
+```ini
+[Service]
```

Comment:
if you provide `etcd_servers` in examples below, put it here too. if you use `https` protocol, probably it worth to mention TLS keypairs described above.